recipe_scrapers/ethanchlebowski.py

```diff
@@ -0,0 +1,41 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class EthanChlebowski(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "ethanchlebowski.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        return self.schema.instructions()
+
+    def ratings(self):
+        return None
+
+    def cuisine(self):
+        return None
+
+    def description(self):
+        return None
```

Comment:
Let's fill in this field with the `description` OpenGraph tag from the page?

(there's some code we could borrow from https://github.com/hhursev/recipe-scrapers/blob/00a460d2c20efe6d17485ca2c3479ea0131693af/recipe_scrapers/sundpaabudget.py#L44 to do the same thing)

Reply:
I was just about to ask you if you knew how to get the description, added this as well

Reply:
Cool, that change looks good.  One of the secrets of writing scrapers is that often much of the code is already implemented for other websites - the trick is finding it :)

(I spend a lot of time using both code search in GitHub, and `grep` at the command-line for this)